src/StoreApi/README.md

```diff
@@ -66,7 +66,9 @@ Available resources in the Store API are listed below, with links to more detail
 |                                                              | `GET`, `POST`, `PUT`, `DELETE` | [`/wc/store/v1/cart/items/:key`](docs/cart-items.md#single-cart-item)                         |
 | [`Cart Coupons`](docs/cart-coupons.md)                       | `GET`, `POST`, `DELETE`        | [`/wc/store/v1/cart/coupons`](docs/cart-coupons.md#list-cart-coupons)                         |
 |                                                              | `GET`, `DELETE`                | [`/wc/store/v1/cart/coupon/:code`](docs/cart-coupons.md#single-cart-coupon)                   |
-| [`Checkout`](docs/checkout.md)                               | `GET`, `POST`                  | [`/wc/store/v1/checkout`](docs/checkout.md)                                                   |
+| [`Checkout`](docs/checkout.md)                               | `GET`, `POST`                  | [`/wc/store/v1/checkout`](docs/checkout.md)     
+| [`Checkout order`](docs/checkout-order.md)                   | `POST`                         | [`/wc/store/v1/checkout`](docs/checkout-order.md)                                             |
+| [`Order`](docs/order.md)                                     | `GET`                          | [`/wc/store/v1/order`](docs/order.md)  
```

Comment:
```suggestion
| [`Order`](docs/order.md)                                     | `GET`                          | [`/wc/store/v1/order/:id`](docs/order.md)  
```
I noticed that for the order endpoint, the path parameter `:id` is also missing. Similar to the checkout order endpoint, we should mention that path parameter here.

Reply:
@nielslange Good catch!  Updated all the above comments here [d8c1eee](https://github.com/woocommerce/woocommerce-blocks/pull/11157/commits/d8c1eee3ef9fe4c9b0e7b46a89123b04b57a03ca). Thank you!

Reply:
>I was wondering if we should flip the sections Responses and Get Order? That way, the example on how to get the order is visible immediately. 

I updated this part, but I want to mention that I followed the cart endpoint. Should we also update the cart endpoint?

Reply:
> @nielslange Good catch! Updated all the above comments here https://github.com/woocommerce/woocommerce-blocks/pull/11157/commits/d8c1eee3ef9fe4c9b0e7b46a89123b04b57a03ca. Thank you!

Thank you for addressing all the suggestions so promptly, @hsingyuc.

Reply:
> [...] I want to mention that I followed the cart endpoint. Should we also update the cart endpoint?

Good catch, @hsingyuc. If you can update the cart endpoint, that'd be helpful.

Additionally, I observed a duplicate TOC in src/StoreApi/docs/checkout.md. Though unrelated to your PR, it'd be great if you could address that too.

However, since these suggestions aren't directly related to your current PR, I've approved it. You can merge now or make my recommended changes first, if possible.

Reply:
Updated cart endpoint doc here [a6ea7a2](https://github.com/woocommerce/woocommerce-blocks/pull/11157/commits/a6ea7a2538ea054fa84253947bab673c03373d79)